lib/update/phase_kubernetes.go

```diff
@@ -406,26 +406,40 @@ func waitForEndpoints(ctx context.Context, client corev1.CoreV1Interface, server
 }
 
 func hasEndpoints(client corev1.CoreV1Interface, labels labels.Set, fn endpointMatchFn) error {
+	services, err := client.Services(metav1.NamespaceSystem).List(
```

Comment:
Had to hack around [this issue](https://github.com/gravitational/gravity.e/issues/3866)